projects/super_res/environment.yaml

```diff
@@ -0,0 +1,29 @@
+name: vsr2
+channels:
+  - pytorch
+  - nvidia
+  - anaconda
+  - conda-forge
+dependencies:
+  - python=3.8
+  - torchvision
+  - pytorch
+  - pytorch-cuda=11.8
+  - notebook
+  - nb_conda_kernels
+  - jupyter_contrib_nbextensions
+  - einops
+  - pillow
+  - tqdm
+  - timm
+  - kornia
+  - accelerate
+  - xarray
+  - seaborn
+  - zarr
+  - google-cloud-sdk
+  - fsspec
+  - gcsfs
+  - joblib
+  - tensorboard
```

Comment:
I think we can remove tensorboard since I'm using wandb

Reply:
Also, could you run python trainer.py without any issues?

Reply:
Right, good catch. Meanwhile, `wandb` isn't there.